.travis.yml

```diff
@@ -7,7 +7,7 @@ cache:
     - $HOME/.cache/pip
 
 addons:
-  postgresql: "9.6"
+  postgresql: "10"
```

Comment:
why not 11?

Reply:
I first tried 11, but, unfortunately, it didn't work because that version is not preinstalled on travis yet.